daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1251,10 +1274,12 @@ void IEEE1588Port::becomeSlave( bool restart_syntonization ) {
 
 	port_state = PTP_SLAVE;
 
-	/*clock->addEventTimer
-		( this, SYNC_RECEIPT_TIMEOUT_EXPIRES,
-		(SYNC_RECEIPT_TIMEOUT_MULTIPLIER*
-		 (unsigned long long)(pow((double)2,getSyncInterval())*1000000000.0)));*/
+	/* TODO : Seems this be active here
```

Comment:
I don't know why we would ever want a sync receipt timer running while we're grandmaster.  Should probably be deleted unless someone knows a good reason for it to continue to stick around as a comment.